### PR TITLE
Enhancement security anonymous only actions

### PR DESCRIPTION
**Pull request by lirantal**
enforcing access to signin/signup resources only to anonymous users so that logged-in users won't be able to access the signin or signup controllers and mess anything up :)

### CODE REVIEW

**Comment by ilanbiala** [comment]
@lirantal is this still necessary?

**Comment by lirantal** [comment]
I haven't yet reviewed Amos changes in 0.4 branch so I can't say. Let's leave it as is until I'm able to peek a little bit on 0.4

File: app/controllers/users/users.authorization.server.controller.js [hunk 1]
```diff
@@ -35,6 +35,23 @@ exports.requiresLogin = function(req, res, next) {
 };
 
 /**
+ * Forces anonymous users to access routes
+ */
+exports.requiresAnonymous = function(req, res, next) {
+    // If user is authenticated, redirect back to referering URL
+    if (req.isAuthenticated()) {
```
Comment (ilanbiala) [inline app/controllers/users/users.authorization.server.controller.js]: What about using `req.user`?

**Comment by ilanbiala** [comment]
@lirantal is this PR still necessary for 0.4.0?

**Comment by lirantal** [comment]
haven't yet reviewed 0.4 in terms of ACLs :(
We can probably close this PR for now and I can submit a new one or re-open later if it will be relevant.

**Pull request closed without merging**